apps/web/screens/Grants/Grant/GrantSidebar.tsx

```diff
@@ -1,15 +1,15 @@
 import { useMemo } from 'react'
 
 import { Box, Button, LinkV2, Stack, Text } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
```

Comment:
Does this work for english? There is a specific another locale context (I think) that the web uses

Reply:
I'll merge for now, and test the localization stuff later.